protected/widgets/menu/assets/menu.css

```diff
@@ -23,61 +23,68 @@
     padding-left: 2em;
 }
 
-
 .menu ul {
     margin: 0;
 }
 
 .menu {
-    overflow: hidden;
-    --animation-duration: 500ms;
     --indicator-width: 3px;
     --indent: 20px;
-    --item-padding-vertical: 10px;
+    --item-padding-vertical: 14px;
+    overflow: hidden;
     padding-top: 10px;
     position: relative;
+    border-radius: 5px;
+    border-top-right-radius: 0px;
     background-color: white;
     grid-area: sidebar;
     font-size: 1em;
     z-index: 10000;
     display: grid;
+    height: 100%;
     grid-template-columns: 1fr;
     grid-template-rows: auto auto auto 1fr auto;
-    grid-template-areas:
-            "image"
-            "title"
-            "separator"
-            "nav"
-            "footer"
+    grid-template-areas: "image" "title" "separator" "nav" "footer";
 }
-
-@keyframes fadein {
-    from { opacity: 0; }
-    to   { opacity: 1; }
+.menu h1 {
+    color: #4c4c4c;
+    text-transform: uppercase;
+    font-size: 18px;
+    letter-spacing: 8px;
+    margin: 0 auto;
+    text-align: center;
+    font-weight: 400;
+}
+.menu img {
+    position: initial;
+    grid-area: image;
+    display: block;
+    max-height: 100px;
+    max-width: 170px;
+    width: 100%;
+    margin: 10px auto;
 }
+
 .menu .footer .left, .menu .footer .right {
     display: inline;
     font-size: 48px;
-    position: absolute;;
+    position: absolute;
     padding: 0;
-    top: calc((100% - 48px - var(--status-height)) / 2);
+    top: 60px;
```

Comment:
This looks wrong to me; the footer of the menu should not be positioned in a fixed distance from the top, right?

Reply:
.left and .right are the arrows in the footer. 
You are right though, they should be auto-centered vertically, if the footer height change

Reply:
This still needs fixing.

Reply:
I've put back the calc